pkg/operator/controller.go

```diff
@@ -708,10 +717,14 @@ func (c *Controller) syncProfile(tuned *tunedv1.Tuned, nodeName string) error {
 		}
 	}
 
+	// Only update Profile if the spec needs to be changed, or !lastUpdateGenSet,
+	// which means we need to clear the profile.Status.Bootcmdline and wait for
```

Comment:
Can we fix this comment since we are no longer clearing `profile.Status.Bootcmdline`?  After that, I believe this should be ready to go.

Reply:
Sorry about that, done now.